CHANGELOG.md

```diff
@@ -1,9 +1,59 @@
+
+This release leverages the Optimizely JavaScript SDK beta5+
+
+This version includes the following changes: 
+
+- New API added to `OptimizelyUserContext`: 
+
+	- `fetchQualifiedSegments()`: this API will retrieve user segments from the ODP server. The fetched segments will be used for audience evaluation. The fetched data will be stored in the local cache to avoid repeated network delays. 
+
+	- When an `OptimizelyUserContext` is created, the SDK will automatically send an identify request to the ODP server to facilitate observing user activities. 
+
+- New APIs added to `OptimizelyClient`: 
+
+	- `sendOdpEvent()`: customers can build/send arbitrary ODP events that will bind user identifiers and data to user profiles in ODP. 
+
+	- `createUserContext()` with anonymous user IDs: user-contexts can be created without a userId. The SDK will create and use a persistent `VUID` specific to a device when userId is not provided. 
+
+For details, refer to our documentation pages:  
+
+- [Advanced Audience Targeting](https://docs.developers.optimizely.com/feature-experimentation/docs/optimizely-data-platform-advanced-audience-targeting)  
+
+- [Client SDK Support](https://docs.developers.optimizely.com/feature-experimentation/v1.0/docs/advanced-audience-targeting-for-client-side-sdks) 
+
+- [Initialize React SDK](https://docs.developers.optimizely.com/feature-experimentation/docs/initialize-sdk-react) 
+
+- [Advanced Audience Targeting segment qualification methods](https://docs.developers.optimizely.com/feature-experimentation/docs/advanced-audience-targeting-segment-qualification-methods-react) 
+
+- [Send Optimizely Data Platform data using Advanced Audience Targeting](https://docs.developers.optimizely.com/feature-experimentation/docs/send-odp-data-using-advanced-audience-targeting-react) 
+
+### Breaking Changes 
+- `OdpManager` in the SDK is enabled by default. Unless an ODP account is integrated into the Optimizely projects, most `OdpManager` functions will be ignored. If needed, `OdpManager` can be disabled when `OptimizelyClient` is instantiated.
+- Updated `murmurhash` dependency to version `2.0.1`.
+- Updated `uuid` dependency to version `8.3.2`.
+- Dropped support for the following browser versions.
+  - All versions of Microsoft Internet Explorer.
+  - Chrome versions earlier than `102.0`.
+  - Microsoft Edge versions earlier than `84.0`.
+  - Firefox versions earlier than `91.0`.
+  - Opera versions earlier than `76.0`.
+  - Safari versions earlier than `13.0`.
+- Dropped support for Node JS versions earlier than `14`.
+
+### Changed
+- Updated `createUserContext`'s `userId` parameter to be optional due to the Browser variation's use of the new `vuid` field ([#229](https://github.com/optimizely/react-sdk/pull/229))
```

Comment:
This seems to be a copy of JS sdk as well. Does the react sdk expose createUserContext() method?